src/organisms/footer/Footer.html

```diff
@@ -1,134 +1,106 @@
-      <a-collapse
-        class="a-footer__links-list"
-        button-class="a-footer__links-button"
-        content-class="a-footer__links-content"
-        v-for="item in menu"
-        :key="item.id"
-        :collapse="!isLargeScreen"
-        :title="item.title"
-        :id="item.id"
-        :disabled="isLargeScreen"
-      >
-        <template #button="{collapseLocal, ariaControls, toggle}">
-            <a-button
-              class="a-footer__links-button a-button--blank a-collapse__button"
-              :aria-controls="!collapseLocal && ariaControls"
-              @click.native="toggle"
-              :aria-expanded="!collapseLocal"
-            >
-              {{ item.title }}
-              <a-icon
-                title="Angle down"
-                icon="angle-down"
-                class="a-collapse__icon"
-              />
-            </a-button>
-          </template>
-          <!-- @slot Links list (Scoped slot) -->
-        <slot
-          name="list"
-          :item="item.items"
-        >
-          <a-list>
-            <a-list-item
-              v-for="link in item.items"
-              :key="link.id"
-            >
-              <a-link
-                class="a-footer__link"
-                :href="link.link"
-              >
-                {{ link.title }}
-              </a-link>
-            </a-list-item>
-          </a-list>
-        </slot>
-      </a-collapse>
-    </section>
-  </div>
+
+    <!--@slot Newsletter (Scoped slot)-->
+    <slot
+      name="newsletter"
+      :submit-newsletter="submitNewsletter"
+    >
+      <a-newsletter
+        class="a-footer__newsletter"
+        :heading="heading"
+        :input="input"
+        button-text="Submit"
+        :checkbox="checkbox"
+        @submit="submitNewsletter"
+      />
+    </slot>
+
+    <a-footer-menu
+      :menu="menu"
+      class="a-footer__menu"
+    />
+  </a-container>
+
   <section class="a-footer__bottom-bar">
-    <div class="a-footer__bottom-bar-handler container">
-      <div class="a-footer__social-handler">
-        <!-- @slot Social media title (Named slot) -->
-        <slot name="socialTitle">
-          <span class="a-footer__social-list-subtitle">
-            {{ socialListSubtitle }}
-          </span>
-        </slot>
-        <a-list class="a-list--horizontal a-footer__social-list">
-          <a-list-item
-            v-for="item in socialMenu"
-            :key="item.id"
-            :aria-label="item.ariaLabel"
-          >
-            <!-- @slot Social media list (Scoped slot) -->
-            <slot
-              name="social"
-              :item="item"
-            >
-              <a-link
-                class="list__icon-link"
-                :href="item.link"
+    <a-container
+      tag="div"
+      class="a-footer__bottom-bar-container"
+    >
+
+      <!-- @slot Social media container (Named slot) -->
+      <slot name="socias">
```

Comment:
Are you sure we can nest slots inside slot?  Please check if it works when we use all slots at once, if not maybe just remove this parent slot `<slot name="socias">`

Reply:
parent slot removed